lib/puppet/util/servicenow.rb

```diff
@@ -96,6 +96,20 @@ def do_snow_request(uri, http_verb, body, user: nil, password: nil, oauth_token:
   end
   module_function :do_snow_request
 
+  def human_readable_event_summary(resource_statuses)
+    summary = ''
+    resource_statuses.values.select { |resource| resource.out_of_sync == true || resource.failed == true }.each do |resource|
+      resource_summary = ''
+      resource.events.each do |event|
+        resource_summary << "#{['', resource.containment_path, "#{event.property}: #{event.message}"].flatten.join('/')}\n"
```

Comment:
Does this output the right thing for pending changes?

Reply:
It mentions that the resource is noop at least, though if you have idea for how you would like to see it made more obvious in the summary output I'm certainly open to it because it doesn't exactly jump out at you.

```
See the PE console for the full report. You can access the PE console at https://loud-transcript.delivery.puppetlabs.net.\n" +

 Resource Statuses:
 /Stage[main]/Main/Node[default]/File[/tmp/pending_corrective_change]/content: current_value '{md5}e6749f1c0f9b9d6893faf97c0398a362', should be '{md5}acbd18db4cc2f85cedef654fccc4a4d8' (noop) (corrective)
   Resource Definition: /etc/puppetlabs/code/environments/production/manifests/site.pp:2
```

Reply:
Ok, cool. That seems good enough.